components/brave_wallet/browser/fil_nonce_tracker.cc

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/fil_nonce_tracker.h"
+
+#include <algorithm>
+#include <memory>
```

Comment:
DITTO, please re-check all header includes in this file, no need to include again for things already included in the header.